src/System.CommandLine.Hosting/System.CommandLine.Hosting.csproj

```diff
@@ -11,6 +11,10 @@
     <DebugType>portable</DebugType>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Why not just exclude this project from the slnf?

Reply:
It is excluded from the .slnf. This is vestigial.